libs/utils/utils.js

```diff
@@ -833,6 +833,16 @@ async function decorateHeader() {
   }
 }
 
+function decorateAnchors(area) {
+  const anchors = area.querySelectorAll('span[class*="icon-anchor-"]');
+  if (anchors.length === 0) return;
```

Comment:
This being utils, maybe we strip it down to
```suggestion
  if (!anchors.length) return;
```